databricks/koalas/series.py

```diff
@@ -50,6 +50,7 @@ def _column_op(f):
     :param self: Koalas Series
     :param args: arguments that the function `f` takes.
     """
+
```

Comment:
It's fine for this PR, but in the future try to minimize unrelated changes.

Reply:
@rxin you mean we use `Series.to_clipboard.__doc__ = DataFrame.to_clipboard.__doc__` ?

Reply:
no i just meant don't add a blank line here since it's unrelated to your change. It's best if a commit is atomic and self-contained about a single feature or bug fix.

Reply:
@rxin sorry, didn't pay attention to blank lines! I'll clean it.